integration/pull_test.go

```diff
@@ -49,6 +49,14 @@ import (
 	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
 )
 
+const (
+	minLayerSize = "20000000"
```

Comment:
nit for follow up: the constant for `minLayerSize` and expected FUSE mounts means that this test could fail unexpectedly if the number or size of layers in the latest image change. We can partially solve this with pinning the version we test  against, but it would still be annoying if tests failed when upgrading the pinned version.

Instead of making these constants, it would be nice if we can choose a size that is exactly the median size of layers in the image and then calculate how many fuse mounts we expect. That way we can be sure it won't start failing after an update and we can test the edge case where min layer size is exactly the size of a layer. I think we can do all this by calling `rebootContainerd` after creating the SOCI index to remove the local state of containerd/the snapshotter.

Can you create an issue for that follow up?

Reply:
https://github.com/awslabs/soci-snapshotter/issues/373